chrome/content/zotero/xpcom/zotero.js

```diff
@@ -2063,3 +2063,79 @@ Zotero.JSON = new function() {
 		return JSON.parse(arg);
 	}
 }
+
+/**
+ * Manages preference panes.
+ */
+Zotero.PreferencePanes = {
```

Comment:
This can be in a separate file, added to zotero-service.js. All files are in a JAR anyway, so better to keep them separate for code manageability.